boefjes/boefjes/plugins/kat_masscan/boefje.json

```diff
@@ -0,0 +1,19 @@
+{
+    "id": "masscan",
+    "name": "masscan",
+    "description": "Quickly scan large amounts of IPs.",
+    "consumes": [
+        "IPV6NetBlock",
```

Comment:
Is massscan fast enough to not die on ipv6 blocks? I'd reccon these would still take far too long

Reply:
@zcrt Do you have any insights on this?

Reply:
To be honest, I have not tested the IPv6 functionality, it seems there is an open issue about on Masscan https://github.com/robertdavidgraham/masscan/issues/7

Reply:
I also don't know if ipv6 works out of the box in docker

Reply:
A quick test indeed seems to result in some docker issues, maybe we should create a separate issue for IPv6 range scanning support?

Reply:
Sounds good, in the meantime i have removed ipv6netblocks from consumes

Reply:
IPv6 is never going to work, an /48 already has 2^80 IP addresses.

Reply:
https://github.com/minvws/nl-kat-coordination/issues/1342